bananas_api/new_upload/session_publish.py

```diff
@@ -152,7 +152,7 @@ def create_package(session):
     version_data = {
         "availability": "new-games",
     }
-    for field in ("upload-date", "md5sum-partial", "filesize", "license", "version"):
+    for field in ("upload-date", "md5sum-partial", "classification", "filesize", "license", "version"):
         version_data[field] = raw_data[field]
```

Comment:
Validate what is written to YAML if `classification` is None. I have only tested with NewGRFs so far, try uploading a non-NewGRF too.